wifiphisher/common/extensions.py

```diff
@@ -367,7 +367,6 @@ def _listen(self):
             dot11.sniff(
                 iface=self._interface,
                 prn=self._process_packet,
-                count=1,
                 store=0,
```

Comment:
I found the main reason is we call sniff for every packet (`count=1`) and the correct way should be used it without the count argument. Is there any reason why we want to use this `count=1` argument?

Reply:
Yes there is a reason for it. If you remove the `count = 1` it does it infinitely and therefore it will never the check the while loop condition.

Reply:
but we have the `stop_filter` argument?

Reply:
then we can remove the while loop and it should be fine.